src/styles/base.scss

```
@@ -1 +1,10 @@
-$violet-color1: #dde2ff;
+$grey-color1: #32394b;
```

Could you please find the best the practices naming scss variable ? `$grey-color-1` doesn't seem too explicit?
```suggestion
$light-grey: #32394b;
```